### ISSUE TITLE
Add list table screen option for actions shown per page

### ISSUE THREAD

**Issue by crstauf** [woocommerce/action-scheduler#601]

**Comment by juliaamosova** [comment]
Hi @crstauf,

Thanks for opening this feature request. I understand that this is in regards to adding an option to `Tools/Scheduled Actions/Screen Options`:

![screen_options](https://user-images.githubusercontent.com/19143190/89843678-6ad92c80-db47-11ea-9b47-a4ce50b44fb6.jpg)

Can you please clarify what option do you have in mind for adding here? Are you referring to adding pagination similar to what is available for the orders for example?

![pagination](https://user-images.githubusercontent.com/19143190/89846980-b93ef900-db50-11ea-9ba5-93112e44b192.jpg)

**Comment by crstauf** [comment]
@juliaamosova Yes, an input box that adjusts the number of actions shown per page in the list table.

**Comment by rrennick** [comment]
The screen options are there but it's missing the `Save` button that you normally find in the screen options.

**Comment by crstauf** [comment]
> The screen options are there

@rrennick On my installation of AS, there is no input field for the "Number of items per page" (also seen in @juliaamosova's screenshots).

**Comment by ovidiul** [comment]
I've added a small PR #771 that enabled pagination on the AS backend (Tools menu).

Any comments or changes are welcomed.
